ou_dedetai/installer.py

```diff
@@ -676,16 +676,8 @@ def ensure_launcher_shortcuts(app=None):
     config.INSTALL_STEPS_COUNT += 1
     ensure_launcher_executable(app=app)
     config.INSTALL_STEP += 1
-    runmode = system.get_runmode()
```

Comment:
We need to verify an install when run from source that its launcher is created correctly. I believe this was set up as a workaround.

Reply:
Reverted the change, doesn't really matter